### ISSUE TITLE
grails install-templates installs only three files

### ISSUE THREAD

**Issue by chrismalan** [grails/scaffolding#16]
Grails 3.1.1.  Only the three files [here](https://github.com/grails3-plugins/scaffolding/tree/master/src/main/templates/scaffolding) are installed.  In the bintray branch at the same url one finds the gsp and a few other files.  

In addition to the above, when I download the gsp files and put them in the right place (src/main/templates/scaffolding), make a small change (change the class="nav" to class="grails-nav") and do a grails generate-all or a grails generate-views, the change I made to the templates is ignored - they are most likely not used at all.

**Comment by chrismalan** [comment]
In addition to the above, when I download the gsp files and put them in the right place (src/main/templates/scaffolding), make a small change (change the class="nav" to class="grails-nav") and do a grails generate-all or a grails generate-views, the change I made to the templates is ignored - they are most likely not used at all.

**Comment by graemerocher** [comment]
Might be due to https://github.com/grails3-plugins/scaffolding/commit/10200c96b8f4307f48cde1b07ff9afdd79979c7c

Thoughts @alvarosanchez

**Comment by alvarosanchez** [comment]
Those templates are definitely part of the web profile now. Maybe we can move them back here, or move `install-template` script to the profile too.

**Comment by chrismalan** [comment]
```
Any idea when that will happen?  I'd
  like to install the changed Scaffolding plugin.

  Thanks.

  On 05/02/16 19:59, Álvaro Sánchez-Mariscal wrote:


  Those templates are definitely part of the web profile now.
    Maybe we can move them back here, or move install-template
    script to the profile too.
  —
    Reply to this email directly or view
      it on GitHub.










-- 
```

Chris Malan
antoniecmalan@gmail.com
You're a card which will have to be dealt with.

**Comment by rlovtangen** [comment]
Dynamic scaffolding also doesn't work after templates being removed from plugin.

**Comment by chrismalan** [comment]
With the new scaffolding 3.2.1 plugin (came out 22/2/2016) the templates are installed using Grails 3.1.1.  However, both generate-all and generate-views ignore these templates.  Hopefully that will be fixed with the next Grails.  These two appears to be Grails scripts and not scaffolding scripts.

**Comment by chrismalan** [comment]
As a follow up to the above, everything works perfectly with Grails 3.0.9 and Scaffolding 3.2.1.  So it does seem as if the problem lies with Grails.

**Comment by graemerocher** [comment]
fixed in 3.2.1

**Comment by ronybarbosa** [comment]
I am using grails 3.0.10.

What is hapenning here is that the artifacts/Service.groovy template is being ignored by both generate-all and create-service.

Do you know if the version 3.2.1 fix it? @graemerocher
